api/src/main/resources/liquibase.xml

```diff
@@ -143,5 +143,56 @@
     		<column name="openconceptlab_updated_on" type="datetime"/>
     	</addColumn>
     </changeSet>
-    
+
+	<changeSet id="openconceptlab-v1-02.11.2016-11.44" author="adamgrzybkowski">
+		<preConditions onFail="MARK_RAN">
+			<foreignKeyConstraintExists foreignKeyName="openconceptlab_item_update"/>
+		</preConditions>
+		<comment>Drop foreign key</comment>
+		<dropForeignKeyConstraint baseTableName="openconceptlab_item" constraintName="openconceptlab_item_update"/>
+	</changeSet>
+
+	<changeSet id="openconceptlab-v1-02.11.2016-11.48" author="adamgrzybkowski">
+		<preConditions>
+			<columnExists tableName="openconceptlab_item" columnName="update_id"/>
+		</preConditions>
+		<comment>Rename update_id column</comment>
+		<renameColumn tableName="openconceptlab_item" oldColumnName="update_id" newColumnName="import_id" columnDataType="bigint"/>
+	</changeSet>
+
+	<changeSet id="openconceptlab-v1-02.11.2016-11.50" author="adamgrzybkowski">
+		<preConditions>
+			<tableExists tableName="openconceptlab_update"/>
+		</preConditions>
+		<comment>Drop update table</comment>
+		<dropTable tableName="openconceptlab_update"/>
+	</changeSet>
+
+	<changeSet id="openconceptlab-v1-02.11.2016-11.51" author="adamgrzybkowski">
+		<preConditions>
+			<not><tableExists tableName="openconceptlab_import"/></not>
+		</preConditions>
+		<comment>Create import table</comment>
+		<createTable tableName="openconceptlab_import">
+			<column autoIncrement="true" name="import_id" type="bigint">
+				<constraints nullable="false" primaryKey="true"/>
+			</column>
+			<column name="uuid" type="char(38)"/>
+			<column name="local_date_started" type="datetime">
+				<constraints nullable="false"/>
+			</column>
+			<column name="local_date_stopped" type="datetime"/>
+			<column name="ocl_date_started" type="datetime" />
+			<column name="error_message" type="varchar(1024)" />
+		</createTable>
+	</changeSet>
+
+	<changeSet id="openconceptlab-v1-02.11.2016-11.52" author="adamgrzybkowski">
+		<preConditions>
+			<tableExists tableName="openconceptlab_import"/>
+		</preConditions>
+		<comment>Add foreign key</comment>
+		<addForeignKeyConstraint constraintName="openconceptlab_item_import" baseTableName="openconceptlab_item" baseColumnNames="import_id" referencedTableName="openconceptlab_import" referencedColumnNames="import_id"/>

```

Comment:
It will fail if there were any previous imports in the db. Imported items would reference non existing imports. I would drop both tables and simply recreate them. The module wasn't used in production anyway.